src/components/KnowledgeGraphPanel.js

```diff
@@ -129,10 +129,18 @@ export default class KnowledgeGraphPanel extends React.Component<KnowledgeGraphP
           const node = this.state.nodes.find((n) => {
             return n.id === nodeId;
           });
-          if (node && node.docId) {
-            this.props.navigateToDoc(node.docId);
-          } else if (node) {
-            this.props.navigateToEntity(node.group, node.label);
+          /**
+           * we do not want any action to be performed
+           * on the double click of the selected node
+           * i.e. the zeroth node.
+           * */
+          if (nodeId !== 0) {
```

Comment:
We could even put this check before line 129. There's no reason to look through the nodes to get the node object if we're never going to use it...

Reply:
True, I thought it is safer to still have to find the selected node but yes we don't want any action to be performed.